<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormFeedDetails</class>
 <widget class="QDialog" name="FormFeedDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="m_tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="m_tpCommon">
      <attribute name="title">
       <string>Common</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="MultiFeedEditCheckBox" name="m_mcbAutoDownloading"/>
         </item>
         <item>
          <widget class="QWidget" name="m_wdgAutoUpdate" native="true">
           <layout class="QFormLayout" name="formLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Auto-downloading of articles</string>
              </property>
              <property name="buddy">
               <cstring>m_cmbAutoUpdateType</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="m_cmbAutoUpdateType">
              <property name="toolTip">
               <string>Select the auto-download strategy for messages of this feed. Default auto-download strategy means that new messges of this feed will be downloaded in time intervals set in application settings.</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="TimeSpinBox" name="m_spinAutoUpdateInterval">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="ArticleAmountControl" name="m_wdgArticleLimiting" native="true"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="MultiFeedEditCheckBox" name="m_mcbSuppressFeed"/>
         </item>
         <item>
          <widget class="QCheckBox" name="m_cbSuppressFeed">
           <property name="text">
            <string>Ignore notifications for this feed</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="MultiFeedEditCheckBox" name="m_mcbDisableFeed"/>
         </item>
         <item>
          <widget class="QCheckBox" name="m_cbDisableFeed">
           <property name="text">
            <string>Disable this feed</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="MultiFeedEditCheckBox" name="m_mcbFeedRtl"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Right-to-left behavior</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="m_cmbRtlBehavior"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimeSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>timespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>MultiFeedEditCheckBox</class>
   <extends>QCheckBox</extends>
   <header>multifeededitcheckbox.h</header>
  </customwidget>
  <customwidget>
   <class>ArticleAmountControl</class>
   <extends>QWidget</extends>
   <header>articleamountcontrol.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_tabWidget</tabstop>
  <tabstop>m_mcbAutoDownloading</tabstop>
  <tabstop>m_cmbAutoUpdateType</tabstop>
  <tabstop>m_spinAutoUpdateInterval</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FormFeedDetails</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>580</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
